jre_emul/android/platform/libcore/ojluni/src/main/java/java/io/OutputStreamWriter.java

```diff
@@ -308,17 +307,15 @@ public void write(int oneChar) throws IOException {
      *             {@code str}.
      */
     @Override
+    //TODO: use StreamEncoder per the update
```

Comment:
In the future, TODO messages take the format "TODO(ldap): comment", so please add your Google ldap. There isn't a need to do so now, since you'll be updating this file soon.

Reply:
Will do. Thanks